Resources/Locale/en-US/mech/mech.ftl

```diff
@@ -17,3 +17,5 @@ mech-energy-missing = Energy: MISSING
 mech-slot-display = Open Slots: {$amount}
 
 mech-no-enter = You cannot pilot this.
+
+mech-eject-pilot-alert = {$user} eject out the pilot {$item}!
```

Comment:
better phrasing here would be "{$user} ejects the pilot of the {$item}!"

Reply:
to be honest since this is the message you get when BEING pulled out, not when you ARE being ejected, it should probably be "{$user} is pulling the pilot out of the {$item}!"